libs/ddd-angular/src/generators/feature/generator.ts

```diff
@@ -3,10 +3,13 @@ import { DddFeatureGeneratorSchema } from './schema';
 import dddFeatureGenerator from '@angular-architects/ddd/src/generators/feature';
 import { applicationGenerator } from '@nrwl/angular/generators';
 import { removeGenerator } from '@nrwl/workspace';
+import { guardValidDomain } from '../../helpers';
 
 const APP_NAME = 'bobby-john-boratos';
 
 export default async function (tree: Tree, options: DddFeatureGeneratorSchema) {
```

Comment:
Why do we need to generate an application just to remove it?
If we're trying to guard against an app not existing, we should create an app-guard, like we did for domains. 🚀

Reply:
Take a look at the `noApp: Boolean` option when calling dddFeatureGenerator.
We could even add it as an option in our schema,

Reply:
yo the noApp boolean is POGGERS 
![dancing-lizard-lizard](https://user-images.githubusercontent.com/22083907/203892443-ed4dce55-a7ac-45f0-8176-a8629c6e8edc.gif)